docs/source/sft_trainer.mdx

```diff
@@ -426,13 +426,13 @@ To use Flash Attention 2, first install the latest `flash-attn` package:
 pip install -U flash-attn
 ```
 
-And add `use_flash_attention_2=True` when calling `from_pretrained`:
+And add `attn_implementation="flash_attention_2"` when calling `from_pretrained`:
```

Comment:
Could you add a documentation section here to explain what `ModelConfig` does and how to use it together with the other utility methods that you have exposed on main init? 🙏

Reply:
Done!